src/components/ha-gauge.ts

```diff
@@ -8,6 +8,10 @@ import { FrontendLocaleData } from "../data/translation";
 import { getValueInPercentage, normalize } from "../util/calculate";
 import { isSafari } from "../util/is_safari";
 
+// Safari version 15.2 and up behaves differently than other Safari versions.
```

Comment:
😠

Reply:
We don't even know this. Maybe 15.2.1 works again :shrug:

Reply:
It might depend on the SVG structure / hierarchy, @thomasloven . My custom SVG based cards didn’t need any change from iOS 14 to iOS 15. No idea why, but they still work 😄